collector/Cargo.toml

```diff
@@ -44,6 +44,7 @@ inquire = "0.7.5"
 
 benchlib = { path = "benchlib" }
 database = { path = "../database" }
+uuid = { version = "1.16.0", features = ["v4"] }
```

Comment:
This shouldn't be needed? The `collector` crate doesn't seem to use `uuid`.

Reply:
Interesting, for some reason the pipeline failed at one point due to `uuid` not existing. I've done a clean build locally and, yes, it can be removed; 6523fc135a89013a571934099e8debb9e9cd313f